observability/monitoring/common-monitoring-configuration-scenarios.adoc

```diff
@@ -0,0 +1,57 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="common-monitoring-configuration-scenarios"]
+= Common monitoring configuration scenarios
+include::_attributes/common-attributes.adoc[]
+:context: common-monitoring-configuration-scenarios
+
+toc::[]
+
+After {product-title} is installed, core platform monitoring components immediately begin collecting metrics, which you can query and view.
+The default in-cluster monitoring stack includes the core platform Prometheus instance that collects metrics from your cluster and the core Alertmanager instance that routes alerts, among other components.
+Depending on who will use the monitoring stack and for what purposes, as a cluster administrator, you can further configure these monitoring components to suit the needs of different users in various scenarios.
+
+In addition to core platform monitoring, you can also optionally xref:../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[enable monitoring for user-defined projects] for user workload monitoring.
+Users can then monitor their own services and workloads without the need for an additional monitoring solution.
+
+[id="configuring-core-platform-monitoring-first-steps_{context}"]
+== Configuring core platform monitoring: First steps
+
+After {product-title} is installed, cluster administrators typically perform certain basic activities to configure core platform monitoring to suit their needs.
```

Comment:
If this is a post-installation activity that customers are expected to do, it might be helpful to add it somewhere in the Post-installation configuration book. I see that we do tell customers to configure alerts and notifications as part of post-installation configuration (which is also mentioned in this assembly in the common scenarios); do we need to expand that to include all of the monitoring configuration scenarios that are highlighted here?

Reply:
That's something to think about, for sure. But it's outside the scope of this PR. I'll write up a separate Jira to explore possible solutions to address this question. Thanks!